app/controllers/state_file/questions/retirement_income_subtraction_controller.rb

```diff
@@ -22,11 +22,11 @@ def self.show?(intake)
       private
 
       def num_items
-        current_intake.state_file1099_rs.count
+        current_intake.eligible_1099rs.count
```

Comment:
Once https://github.com/codeforamerica/vita-min/pull/5643 is merged, we should remove ` num_items` defined in `app/controllers/state_file/questions/id_retirement_and_pension_income_controller.rb`

Reply:
You got it